inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -242,22 +246,16 @@ class LiteratureForm(INSPIREForm):
         export_key='categories',
     )
 
-    # isbn = ISBNField(
-    #     label=_('ISBN'),
-    #     widget_classes='form-control',
-    # )
-
     import_buttons = fields.SubmitField(
         label=_(' '),
         widget=import_buttons_widget
     )
 
     types_of_doc = [("article", _("Article/Conference paper")),
-                    ("thesis", _("Thesis"))]
-
-    # ("chapter", _("Book Chapter")),
-    # ("book", _("Book")),
-    # ("proceedings", _("Proceedings"))]
+                    ("thesis", _("Thesis")),
+                    ('book', _('Book')),
+                    ('chapter', _('Book chapter'))]
+    # ("proceedings", _("Proceedings"))
```

Comment:
Try not to leave commented code without a comment or a pointer to an issue that explains what's missing here.